src/main/java/seedu/address/model/person/Pin.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.person;
+
+public class Pin {
+    public static final String MESSAGE_CONSTRAINTS = "Pin status should be either true or false";
+    public static final String PINNED_STRING = "true";
+    public static final String NOT_PINNED_STRING = "false";
+
+    private final boolean isPinned;
+
+    /**
+     * Constructs a pin.
+     *
+     * @param isPinned if pinned or not.
+     */
+    public Pin(boolean isPinned) {
+        this.isPinned = isPinned;
+    }
+
+    /**
+     * Constructs a pin based on the pin status.
+     *
+     * @param pinString string indicating if pinned or not.
+     */
+    public Pin(String pinString) {
+        assert pinString.equals(PINNED_STRING) || pinString.equals(NOT_PINNED_STRING)
+                : "Pinned string not saved correctly";
+        isPinned = pinString.equals(PINNED_STRING);
```

Comment:
The constructor design deviates from the rest of the model attributes. Can I clarify why this is different?